app/assets/javascripts/dragdrop.js

```diff
@@ -17,44 +17,36 @@ jQuery(function() {
     );
   }
 
-  $('.dropzone').on('dragover',function(e) {
-    e.preventDefault();
-    $('.dropzone').addClass('hover');
-  });
-  $('#side-dropzone').on('dragover',function(e) {
-    e.preventDefault();
-    $('#side-dropzone').addClass('hover');
-  });
-  $('.dropzone').on('dragout',function(e) {
-    $('.dropzone').removeClass('hover');
-  });
-  $('#side-dropzone').on('dragout',function(e) {
-    $('#side-dropzone').removeClass('hover');
-  });
-  $('.dropzone').on('drop',function(e) {
-    // this 'drop' listener is also reused for pages with just one form, ie. /wiki/new
-    const closestCommentFormWrapper = e.target.closest('div.comment-form-wrapper'); // this returns null if there is no match
-    e.preventDefault();
-    let params = {};
-    // there are no .comment-form-wrappers on /wiki/edit or /wiki/new
-    // these pages just have a single text-input form.
-    if (closestCommentFormWrapper) {
-      $D.selected = $(closestCommentFormWrapper);
-      // assign the ID of the textarea within the closest comment-form-wrapper
-      params['textarea'] = closestCommentFormWrapper.querySelector('textarea').id;
-    } else {
-      // default to #text-input
-      // ideally there should only be one per page
-      params['textarea'] = 'text-input';
-    }
-    $E.initialize(params);
-  });
-  $('#side-dropzone').on('drop',function(e) {
-    e.preventDefault();
-  });
-
   // these functions are also used on /wiki/edit and /wiki/new pages
   $('.dropzone').each(function() {
+    $(this).on('dragenter',function(e) {
+      e.preventDefault();
+      $(e.currentTarget).addClass('hover');
+    });
+
```

Comment:
Changed `dragover` and `dragout` to `dragenter` and `dragleave` so they don't fire every time the user makes a movement with the image on the surface of the div.